ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTY2OTYxNDc1LjE3NTMyNjk1NTAifSwiYW5vbklkIjoiMTY3NDhkYjE5MWQzY2VmZSJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL21vZGVscy9jb3VwZS9jbGEvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjgiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIxNDM4NjE4ODgzODA4MDEzOCIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=138.0.7204.157)
Stacktrace:
#0 0x559e3f8b27da <unknown>
#1 0x559e3f3572e0 <unknown>
#2 0x559e3f33e8e6 <unknown>
#3 0x559e3f33d1a0 <unknown>
#4 0x559e3f33d92f <unknown>
#5 0x559e3f33d884 <unknown>
#6 0x559e3f35b50f <unknown>
#7 0x559e3f400b0a <unknown>
#8 0x559e3f3ce8b2 <unknown>
#9 0x559e3f3f471c <unknown>
#10 0x559e3f3ce683 <unknown>
#11 0x559e3f39ab5b <unknown>
#12 0x559e3f39bf31 <unknown>
#13 0x559e3f87772b <unknown>
#14 0x559e3f87b534 <unknown>
#15 0x559e3f85e229 <unknown>
#16 0x559e3f87c0d8 <unknown>
#17 0x559e3f84261f <unknown>
#18 0x559e3f89fdd8 <unknown>
#19 0x559e3f89ffb6 <unknown>
#20 0x559e3f8b1af6 <unknown>
#21 0x7f1807c9caa4 <unknown>
#22 0x7f1807d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAxMTgzIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJpdF9JVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2NvdXBlL2NsYS9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2OCJ9LCJmbGFncyI6eyJwYWdlVmlldyI6dHJ1ZX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTY2OTYxNDc1LjE3NTMyNjk1NTAifSwiYW5vbklkIjoiMTY3NDhkYjE5MWQzY2VmZSJ9LCJwZXJmb3JtYW5jZSI6eyJzZGtQYXJzZVRpbWUiOjF9LCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9LCJjYXRhbG9nIjp7IlByb2R1Y3QiOnsiX2lkIjoiMTE4MyIsImRpbWVuc2lvbnMiOnsiQmF1bXVzdGVyU2hvcnQiOlsiMTE4MyJdfX19LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjI0NDM4NDc4OTY4OTkyOTcyIn0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=138.0.7204.157)
Stacktrace:
#0 0x559e3f8b27da <unknown>
#1 0x559e3f3572e0 <unknown>
#2 0x559e3f33e8e6 <unknown>
#3 0x559e3f33d1a0 <unknown>
#4 0x559e3f33d92f <unknown>
#5 0x559e3f33d884 <unknown>
#6 0x559e3f35b50f <unknown>
#7 0x559e3f400b0a <unknown>
#8 0x559e3f3ce8b2 <unknown>
#9 0x559e3f3f471c <unknown>
#10 0x559e3f3ce683 <unknown>
#11 0x559e3f39ab5b <unknown>
#12 0x559e3f39bf31 <unknown>
#13 0x559e3f87772b <unknown>
#14 0x559e3f87b534 <unknown>
#15 0x559e3f85e229 <unknown>
#16 0x559e3f87c0d8 <unknown>
#17 0x559e3f84261f <unknown>
#18 0x559e3f89fdd8 <unknown>
#19 0x559e3f89ffb6 <unknown>
#20 0x559e3f8b1af6 <unknown>
#21 0x7f1807c9caa4 <unknown>
#22 0x7f1807d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjE2Njk2MTQ3NS4xNzUzMjY5NTUwIn0sImFub25JZCI6IjE2NzQ4ZGIxOTFkM2NlZmUifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvY291cGUvY2xhL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjY4IiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6MX0sIl90b29sc0V2ZW50TGlua0lkIjoiNTM2MzM2MjUyMDE3NDE3NiIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=138.0.7204.157)
Stacktrace:
#0 0x559e3f8b27da <unknown>
#1 0x559e3f3572e0 <unknown>
#2 0x559e3f33e8e6 <unknown>
#3 0x559e3f33d1a0 <unknown>
#4 0x559e3f33d92f <unknown>
#5 0x559e3f33d884 <unknown>
#6 0x559e3f35b50f <unknown>
#7 0x559e3f400b0a <unknown>
#8 0x559e3f3ce8b2 <unknown>
#9 0x559e3f3f471c <unknown>
#10 0x559e3f3ce683 <unknown>
#11 0x559e3f39ab5b <unknown>
#12 0x559e3f39bf31 <unknown>
#13 0x559e3f87772b <unknown>
#14 0x559e3f87b534 <unknown>
#15 0x559e3f85e229 <unknown>
#16 0x559e3f87c0d8 <unknown>
#17 0x559e3f84261f <unknown>
#18 0x559e3f89fdd8 <unknown>
#19 0x559e3f89ffb6 <unknown>
#20 0x559e3f8b1af6 <unknown>
#21 0x7f1807c9caa4 <unknown>
#22 0x7f1807d29c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:21 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Perso-MB-PC/QA-Perso-MB-PC/Tests/BFV1_attempt_2.png'
WARNING  root:ImageVerifier.py:56 ❌ Image not found in the specified selector. Expected src: /content/dam/hq/personalization/campaignmodule/